libs/admin/src/lib/state/admin.effects.ts

```diff
@@ -0,0 +1,118 @@
+import { inject, Injectable } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { fromAppShared } from '@asset-sg/client-shared';
+import { isDecodeError } from '@asset-sg/core';
+import * as RD from '@devexperts/remote-data-ts';
+import { UntilDestroy } from '@ngneat/until-destroy';
+import { Actions, createEffect, ofType } from '@ngrx/effects';
+import { Store } from '@ngrx/store';
+import * as D from 'io-ts/Decoder';
+import { filter, map, merge, switchMap, tap } from 'rxjs';
+
+import { AdminService, User, Workgroup } from '../services/admin.service';
+import * as actions from './admin.actions';
+import { AdminState, AppStateWithAdmin } from './admin.reducer';
+
+@UntilDestroy()
+@Injectable()
+export class AssetAdminEffects {
```

Comment:
We can just call this `AdminEffects`, I don't think the prefix adds anything of value.
```suggestion
export class AdminEffects {
```